include/matx/generators/random.h

```diff
@@ -54,6 +54,28 @@ __global__ void curand_setup_kernel(Gen *states, uint64_t seed, index_t size)
 };
 #endif
 
+
+/**
+ * @brief Get a random number
+ *
+ * @tparam T inner type of data
+ * @tparam Gen Generator type
+ * @param val Value to store in
+ * @param state Generator state
+ * @param min min of the range to generate
+ * @param max max of the range to generate
+ */
+template <typename T, typename Gen>
+__inline__ __MATX_DEVICE__ void get_randomi(T &val, Gen *state, double min, double max)
```

Comment:
I thought you said curand_uniform returns a number between 0 and FLOAT_MAX, but it looks like it's 0..1?